README.md

```diff
@@ -9,12 +9,44 @@ a policy. For example, you should be able to generate reports of all kinds for
 your auditors, get alerts, and create playbooks to fix your hosts.
 
 
-## Getting started
+## Architecture
 
 This project does two main things:
 
-1 - Connect to a Kafka message queue [provided by the Insights Platform](https://github.com/RedHatInsights/insights-upload)
-2 - Serve as the API backend for the web UI [compliance-frontend](https://github.com/RedHatInsights/compliance-frontend) and for other consumers.
+1. Serve as the API/GraphQL backend for the web UI
```

Comment:
GraphQL is an API. If you want to differentiate them, we can mention REST vs. GraphQL.

Reply:
Differentiated below.